### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by luftedar**
-  [x] *JavaScript best practices.*
-  [x] *DRY, KISS, and YAGNI rules.*
-  [x] *HTML and CSS best practices.*

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
```
Comment (sja-thedude) [inline dist/index.html:8]: Instead of adding the font family link in html you can add it in CSS file for neat code.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi Orcun!
You have done a great job! 👏 
There are still some parts of the code you have to work with.

Kindly add a .gitignore file for node_modules!
You can use foreach instead of for loop in your files 
https://github.com/luftedar/todolist-review/blob/388abfa57da683973ce51abc17129922e6c11786/src/index.js#L32
https://github.com/luftedar/todolist-review/blob/388abfa57da683973ce51abc17129922e6c11786/src/interactive.js#L8
for better code. 

Happy Coding!!! 👍

**Comment by luftedar** [comment]
## Hi SJA. Thanks for your review. :100: 

I've implemented your requests :+1: 

> I'm glad for your precious review. Have a nice coding

File: dist/index.html (hunk 1 above)
Comment (sja-thedude) [reply dist/index.html:8]: Good

**Review by sja-thedude — APPROVED** [review]
Hi Orcun!! 🥂 

Great :muscle:  job :raised_hands:  implementing all of the requirements for this project.

# Status: Approved 👍 :partying_face: 
Your project has been approved.

## Congratulations :+1:  :fireworks: 

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**